node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -501,7 +501,8 @@ func NewGetDmsPathHelperGeneric(executer executer.ExecuterInterface) GetDmsPathH
 
 func (o GetDmsPathHelperGeneric) WaitForDmToExist(volumeUuid string, maxRetries int, intervalSeconds int) (string, error) {
 
-	args := []string{"show", "maps", "raw", "format", "\"", "%d" + MpathdSeparator + "%w", "\""}
+	formatOrder := strings.Join([]string{"%d", "%w"}, MpathdSeparator)
```

Comment:
IMO it's not just about order, but also which fields to show.
maybe just `format`?

Reply:
maybe `formatTemplate`?

Reply:
sure

Reply:
done